src/components/CreateToken/TokenCreatedTx.tsx

```diff
@@ -0,0 +1,107 @@
+import React, { useState, useEffect } from 'react';
+import { useSelector } from 'react-redux';
+import { dispatch } from 'store/rematch';
+import styled from '@emotion/styled';
+
+import {
+  selectModalOptions,
+  selectTokenCreationTxError,
+  selectTokenCreationTxId,
+  selectTokenCreationTxStatus,
+} from 'store/selectors';
+import { DEFAULT_NULL_MODAL } from 'store/models/environment';
+import { ViewType } from 'vars/defines';
+import { V } from 'util/theming';
+import TokenType from 'util/types/TokenType';
+
+import ExplorerLink from 'components/_General/ExplorerLink';
+import { Button } from 'components/_General/buttons';
+import Loader from 'components/_General/Spinner';
+
+const Title = styled.h2<{ success: boolean }>`
+  color: ${props => (props.success ? V.color.growth : V.color.danger)};
+  margin-top: 0;
+`;
+
+const TokenCreatedTx: React.FC = () => {
+  const { tokenTypeName, tokenTypeNameCapitalized } = useSelector(selectModalOptions) as {
+    tokenType: TokenType;
+    tokenTypeName: string;
+    tokenTypeNameCapitalized: string;
+  };
+
+  const error = useSelector(selectTokenCreationTxError);
+  const txStatus = useSelector(selectTokenCreationTxStatus);
+  const txId = useSelector(selectTokenCreationTxId);
+
+  const [isBroadcasting, setIsBroadcasting] = useState(true);
+  const [hasError, setHasError] = useState(false);
+
+  const closeModal = () => dispatch.environment.SET_MODAL(DEFAULT_NULL_MODAL);
+  const goToWallet = () => {
+    dispatch.environment.SET_VIEW(ViewType.DASHBOARD);
+    closeModal();
+  };
+
+  useEffect(() => {
+    setIsBroadcasting(txStatus === 0);
+    setHasError(txStatus < 0);
+  }, [txStatus]);
+
+  useEffect(() => {
+    return () => {
+      dispatch.tokenCreation.RESET_TX();
+    };
+  }, []);
+
+  if (isBroadcasting) {
+    return (
+      <div css={{ textAlign: 'center' }}>
+        <Loader bgColor={V.color.modal.bg} />
+        <p>Broadcasting transaction...</p>
+      </div>
+    );
+  }
+
+  return (
+    <div>
+      <Title success={!hasError}>
+        {hasError ? `Failed to create ${tokenTypeName}` : `${tokenTypeNameCapitalized} created!`}
+      </Title>
+
+      {hasError ? (
+        <>
+          <p>
+            An error has ocurred while broadcasting your {tokenTypeName}: {error}.
+          </p>
+          <p>
+            Please confirm that no transaction has been broacast and try again in a few minutes.
```

Comment:
how will they check? We could add a suggestion, aka check your address txs on explorer and add a link for users - https://explorer.tokel.io/address/RPVRY4T2KkYN6k9AmRCXFJQcUYmpBYB83Q